.idea/HelloWorld.java

```diff
@@ -0,0 +1,6 @@
+public class HelloWorld {
+   public static void main(String[] args) {
+      // Prints "Vegas Curry" in the terminal window.
+      System.out.println("Vegas Curry");
+   }
```

Comment:
Good job Vegas.  I like to commenting that describes the code you are writing.  This is a great habit to get into.